src/typed-method-types/workflows/triggers/scheduled.ts

```diff
@@ -29,9 +29,10 @@ type BaseFrequencyType = {
   /** @description How often the trigger will repeat, respective to the frequency type */
   repeats_every?: number;
   /**
-   * @description The nth week of the chosen frequency type (not available for daily or weekly triggers)
-   * @example The 3rd week of the month */
-  on_week_num?: number;
+   * @description The nth week of the chosen frequency type (not available for daily, weekly, or yearly triggers)
```

Comment:
Not sure if it's entirely clear from the description that "-2" would not be a valid entry. I imagine someone might try to use -2 for the second last week of the month

Reply:
Yeah, `-2` could be the second to last week in the month, but the options in this union type are specified by the backend. If a developer tries to use `on_week_num: -2` they'll get a type error